pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -3603,3 +3703,16 @@ func lookupCapabilities(ctx *domainContext) (*types.Capabilities, error) {
 	}
 	return &capabilities, nil
 }
+
+func getnodeNameAndUUID(ctx *domainContext) error {
```

Comment:
```suggestion
func (ctx *domainContext) retrieveNodeNameAndUUID(ctx *domainContext) error {
```
as this is not a getter method

Reply:
same here: https://github.com/lf-edge/eve/pull/4407/files#diff-6ac92d68c9dd6b3a3052cf0959aa71795adcabc49ca12faae647c9fe27f6340dR1708

Reply:
Ok.